### ISSUE TITLE
` setup/start_ckan_development.sh.override` is outdated

### ISSUE THREAD

**Issue by roll** [ckan/ckan-docker#129]
# Overview

The working version is https://github.com/ckan/ckan-docker-base/blob/main/ckan-2.10/dev/setup/start_ckan_development.sh

**Comment by wardi** [comment]
Maybe better to delete the file from this repo instead of copying the latest again?

**Comment by kowh-ai** [comment]
Yes it may be best to delete these (`*.override` files in this repo) and add instructions to the README how/when to copy files from the [ckan-docker-base](https://github.com/ckan/ckan-docker-base) so that they are overidden from within the ckan-docker repo...

**Comment by wardi** [comment]
I found this README instruction/example a little confusing because you *also* have to remember to change your Dockerfile. Why not make it automatic if we have a special directory for it?

**Comment by roll** [comment]
Thanks a lot for looking at it!

My 5 cents about DX as I just worked on an extension using `ckan-docker`:
- the most stupid mistake I made was changing `CKAN_SITE_URL` to `https://ckan:5000` -- it took a few hours to figure out why I was getting `bad request` errors from flask. It might make sense to remove `https` from defaults or add a note
- you need to change `datapusher` callback urls to `http://ckan-dev:...` -- not really easy to figure out without at least some ckan/compose knowledge
- I also sent a PR to support `pyproject.toml` - https://github.com/ckan/ckan-docker-base/pull/48
